src/core/ContentWindow.cpp

```diff
@@ -166,6 +169,38 @@ bool ContentWindow::isHidden() const
     return windowState_ == HIDDEN;
 }
 
+void ContentWindow::setSelected( bool value )
```

Comment:
Yes, your second solution is more compact, I can do that. I added these functions for because I needed to call them from QML, and I didn't know how to use enum values as a parameter. But they can certainly be exported as well...